### DISCUSSION TITLE
On Proxmox, why do I need to encrypt the data drive?

### DISCUSSION THREAD

**Discussion by pl804** [noahbliss/mortar#39] [Q&A] [answered]
If I already selected LVM with encryption during the Debian install?

**Comment by noahbliss** [comment, marked as answer]
The answer is in the question. Proxmox by default uses a separate partition/drive for data which isn't protected by LUKS. If you decide to just use a big root partition, you're right this wouldn't be needed. But if you use unencrypted disks for data, you can "extend" your encryption chain of trust by doing file based encryption on them with the key on the proxmox root drive with restrictive permissions.

**Reply by pl804** [reply]
Ah, thanks. I plan to use the rest of the drive with the OS as local-over, then create another storage pool from 2 NVMEs (that are outside of the RAID the OS sits on). Should I go through Step 6 first before Step 5?

**Comment by noahbliss** [comment]
I'd still do the written order. Step 6 is just an example way of protecting the data disk, but you can definitely use other methods.
